src/components/Panel/Panel.jsx

```diff
@@ -49,22 +49,39 @@ const Panel = React.createClass(createLucidComponentDefinition({
 	},
 
 	render: function() {
-		const {className, style, children, isGutterless} = this.props;
+		const {
+			className,
+			style,
+			children,
+			isGutterless
```

Comment:
I agree, though we should change it on bother `Grid` and here, and technically it's a breaking change.